### DISCUSSION TITLE
Example INO with ESP_WiFiManager with HTTPClient using https

### DISCUSSION THREAD

**Discussion by glfran35** [khoih-prog/ESP_WiFiManager#101] [Q&A]
I've been able to get the basic examples for ESP_WiFiManager to work, but when adding functions to create a https client, like using ESP8266HTTPClient, it fails to connect.  if there something overlook in the document?  thanks for the pointers in the right direction.

**Comment by khoih-prog** [comment]
Hi @glfran35 

This is not designed for HTTPS. But if you 're interested, you can modify the library yourself to support `HTTPS / SSL`. But I anyway really don't see the necessity to use SSL for `Config Portal` in the local network.

Good Luck,

**Comment by glfran35** [comment]
Thanks for the reply.. but I think the message did not define the issue.


When I leverage the INO example: AutoConnectWithFeedBack.ino
Then try to add code from BasicHTTPSClient.ino to issue an HTTPS URL..

The Serial. Displays  [HTTPS] GET... failed, error: connection failed

If BasicHTTPSClient.ino  is run as is with SIDD and password.. the example URL works.

The ask was if there was an example INO where in it runs the portal and then creates a HTTPS client request to an external site?

Thanks for a guidance.

**Reply by khoih-prog** [reply]
The library only takes care of the config portal, and won't intervene in normal operation after Config-Portal exits.

Sorry **this has nothing to do to this library.** You have to know how to correctly use HTTPS and merge with this library code. No plan to add that kind of examples.

Ask on **Arduino Forum**, certainly with full [**MRE**](https://stackoverflow.com/help/minimal-reproducible-example) and understandable question for help.
